exercises/xorcism/.meta/description.md

```diff
@@ -0,0 +1,69 @@
+Write a streaming adaptor which contains a reference to a key, and bitwise-xors it with arbitrary data.
```

Comment:
While reading this sentence, I found that https://doc.rust-lang.org/std/iter/trait.Iterator.html contains both "adaptor" and "adapter". Both are valid spellings, but if that Iterator documentation had been consistently using one spelling or the other, I would have suggested using whichever spelling matches the documentation. Since they use both, it seems the choice is ours.

Reply:
Thanks for checking that. Agree that if the documentation were consistent, we should remain consistent with the documentation; as it stands, I prefer the 'or' variant over 'er'.